docs/lang/articles/get-started/index.md

```diff
@@ -5,7 +5,40 @@ slug: /
 
 # Getting Started
 
-Welcome to the Taichi Language documentation!
+## Target audience
+
+End users who *only* wish to quickly set up Taichi for simulation or high-performance numerical computation.
+
+:::caution IMPORTANT
+
+For developers who are interested in the compiler, computer graphics, or high-performance computing, and would like to contribute new features or bug fixes to the [Taichi programming language](https://github.com/taichi-dev/taichi), see the [Developer installation](../contribution/dev_install.md) for more information on building Taichi from source.
+
+:::
+
+## Prerequisites
+
+### Python
+
+3.6/3.7/3.8/3.9/3.10 (64-bit)
+
+:::note
+
+Taichi recommends installing Python from [Miniforge](https://github.com/conda-forge/miniforge/#download) conda if you are on a MacBook with M1 chip.
+
+:::
+
+### Supported systems and backends
+
+The following table lists the supported operating systems and the backends that Taichi supports on these platforms:
+
+| **platform** |      **CPU**       |      **CUDA**      |     **OpenGL**     |     **Metal**      |    **C source**    |
```

Comment:
Should we add Vulkan and remove C source here?

Reply:
Agree, the Vulkan backend works smoothly.